apps/daimo-mobile/src/view/screen/onboarding/InvitePage.tsx

```diff
@@ -0,0 +1,184 @@
+import {
+  DaimoInviteStatus,
+  DaimoLinkInvite,
+  DaimoLinkNoteV2,
+  DaimoNoteState,
+  DaimoNoteStatus,
+  EAccount,
+  formatDaimoLink,
+  getEAccountStr,
+} from "@daimo/common";
+import { DaimoChain } from "@daimo/contract";
+import Octicons from "@expo/vector-icons/Octicons";
+import * as Clipboard from "expo-clipboard";
+import { useEffect, useState } from "react";
+import { Linking, View, StyleSheet } from "react-native";
+
+import { OnboardingHeader } from "./OnboardingHeader";
+import { getInvitePasteLink } from "../../../logic/invite";
+import { useFetchLinkStatus } from "../../../logic/linkStatus";
+import { ButtonBig, TextButton } from "../../shared/Button";
+import { InputBig, OctName } from "../../shared/InputBig";
+import { IntroTextParagraph } from "../../shared/IntroTextParagraph";
+import Spacer from "../../shared/Spacer";
+import { color } from "../../shared/style";
+import { TextCenter, TextLight } from "../../shared/text";
+
+export function InvitePage({
+  onNext,
+  onPrev,
+  daimoChain,
+  inviteLink,
+  setInviteLink,
+}: {
+  onNext: ({ isTestnet }: { isTestnet: boolean }) => void;
+  onPrev?: () => void;
+  daimoChain: DaimoChain;
+  inviteLink: DaimoLinkInvite | DaimoLinkNoteV2 | undefined;
+  setInviteLink: (
+    inviteLink: DaimoLinkInvite | DaimoLinkNoteV2 | undefined
+  ) => void;
+}) {
+  // Handle invite links from deep link opens.
+  const [text, setText] = useState(
+    inviteLink ? formatDaimoLink(inviteLink) : ""
+  );
+  const [isValid, setIsValid] = useState(!!inviteLink);
+  const [sender, setSender] = useState<EAccount | undefined>(undefined);
+
+  // HACK: Workaround for testnet to never query API
```

Comment:
long term we can fix this by merging the APIs.
single API > can multiplex on backend

Reply:
pulled into its own issue